locations/spiders/yettel_bg.py

```diff
@@ -10,20 +14,45 @@ class YettelBGSpider(Spider):
         "brand_wikidata": "Q14915070",
         "country": "BG",
     }
-    start_urls = ["https://www.yettel.bg/store-locator/json"]
+    start_urls = ["https://www.yettel.bg/dA/1d9e589cca"]
```

Comment:
The spreadsheet has already changed to ``.

I recommend starting with the store finder and extracting the url from there:
```suggestion
    start_urls = ["https://www.yettel.bg/faq/digital-customer-service/store-locator"]
```